tractolearn.egg-info/PKG-INFO

```diff
@@ -0,0 +1,86 @@
+Metadata-Version: 2.1
```

Comment:
All these files should not have been committed. You may want to add a rule if it does not exist in the `.gitignore` file.

Reply:
Done